crates/cranelift/src/wasmfx/optimized.rs

```diff
@@ -1385,12 +1383,21 @@ pub(crate) fn translate_resume<'a>(
         let mut vmcontref = tc::VMContRef::new(resume_contref, env.pointer_type());
 
         let revision = vmcontref.get_revision(env, builder);
-        let evidence = builder.ins().icmp(IntCC::Equal, revision, witness);
-        emit_debug_println!(env, builder, "[resume] contobj = {:p}, resume_contref = {:p} witness = {}, revision = {}, evidence = {}", contobj, resume_contref, witness, revision, evidence);
-
-        builder
-            .ins()
-            .trapz(evidence, ir::TrapCode::ContinuationAlreadyConsumed);
+        if !cfg!(feature = "unsafe_disable_continuation_linearity_check") {
```

Comment:
I find the following slightly easier to read as it doesn't mix the languages.
```suggestion
        if cfg!(not(feature = "unsafe_disable_continuation_linearity_check")) {
```